tests/e2e/standard/enketo/pregnancy.wdio-spec.js

```diff
@@ -0,0 +1,143 @@
+const utils = require('../../../utils');
+const gatewayApiUtils = require('../../../gateway-api.utils');
+const commonPage = require('../../../page-objects/common/common.wdio.page');
+const loginPage = require('../../../page-objects/login/login.wdio.page');
+const userFactory = require('../../../factories/cht/users/users');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const newPersonPage = require('../../../page-objects/forms/new-person.wdio.page');
+const contactPage = require('../../../page-objects/standard/contacts/contacts.wdio.page');
+const reportsPage = require('../../../page-objects/reports/reports.wdio.page');
+const analyticsPage = require('../../../page-objects/analytics/analytics.wdio.page');
+const genericForm = require('../../../page-objects/forms/generic-form.wdio.page');
+const pregnancyForm = require('../../../page-objects/standard/forms/pregnancy.wdio.page');
+
+describe('New pregnancy', () => {
+  const places = placeFactory.generateHierarchy();
+  const healthCenter = places.find(place => place.type === 'health_center');
+  const user = userFactory.build({ place: healthCenter._id, roles: ['district_admin'] });
+  const pregnantWoman1 = 'Woman1';
+  const pregnantWoman2 = 'Woman2';
+
+  before(async () => { 
+    await utils.saveDocs([...places]);
```

Comment:
```suggestion
    await utils.saveDocs(places);
```

Maybe I am missing something here, but is there a reason to create a new array before passing it to `saveDocs`?

Reply:
You are right, there is no need to create a new array. I changed it in the three files.
Thanks!